app/types.ts

```diff
@@ -0,0 +1 @@
+export type Locale = "en-US" | "es";
```

Comment:
be consistent and use either short codes (two letters) or language and country together. Since you only have one English I'd recommend using only short codes.
```suggestion
export type Locale = "en-US" | "es";
```
```suggestion
export type Locale = "en" | "es";
```